src/Layout/Presentation/ConnectedPresentation.tsx

```diff
@@ -15,6 +15,7 @@ const mapStateToProps = (state: RootState): Partial<PresentationProps> => {
         disproportionalityIndex: state.presentationMenuState.disproportionalityIndex,
         showComparison: state.presentationMenuState.showComparison,
         year,
+        algorithm: state.computationState.algorithm,
```

Comment:
Why was this change necessary? Because we are now dependent on knowing the algorithm to handle logic further down in more deeply nested components properly?

Reply:
Yes, we do not want to calculate closest seat when using largest fraction, Celius does not calculate that either.